metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/resolver/ClasspathFunctionResolver.java

```diff
@@ -65,24 +72,48 @@
  *
  */
 public class ClasspathFunctionResolver extends BaseFunctionResolver {
```

Comment:
If there is never case where we might need to use the older mechanism, then I'd be onboard.